manifests/0000_50_cluster-platform-operator-manager_04-rukpak-core.deployment.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         - --tls-cert-file=/etc/pki/tls/tls.crt
         - --tls-private-key-file=/etc/pki/tls/tls.key
         - --upstream-ca-file=/etc/configmaps/operator-cert-ca-bundle/service-ca.crt
-        image: quay.io/brancz/kube-rbac-proxy:v0.12.0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
What repo is this image from? Can we use a downstream version? Thanks!

Reply:
Yeah, we can use a downstream image here. For context, we're effectively just re-using what the upstream rukpak project is using for images, and the manifests/image-references file that was introduced will substitute those defined images with whatever's tagged in the release payload.

I'd like to keep the scope of these changes small, and using a downstream image here would be a no-op outside of people manually installing these components via kubectl, vs. the supported workflow that relies on the CVO to install/manage/etc. these components.